app/src/main/java/org/nieghborhoodbikeworks/nbw/SharedViewModel.java

```diff
@@ -26,18 +26,24 @@
     private Task mSignInTask;
     private FirebaseDatabase mDatabase = FirebaseDatabase.getInstance();
     private DatabaseReference mUserDatabase = mDatabase.getReference().child("users");
-    private LinkedList<FirebaseUser> mQueue = new LinkedList<>();
-
-    public DatabaseReference getUserDatabase() {
-        return mUserDatabase;
-    }
-    private MutableLiveData<LinkedList<FirebaseUser>> liveData;
+    private List<String> mQueue = new ArrayList<String>();
```

Comment:
Shouldn't this be `ArrayList<String>`? I'm not sure if Java complains about this but if it doesn't then it's all good.